taccsite_cms/templates/guides/data_transfer.html

```diff
@@ -0,0 +1,537 @@
+      </ul>
+
+      <h4 id="local_data_path">
+        Determining the Path to Your Data File(s) on Your Local System
+      </h4>
+
+      <p>
+        In order to transfer your project data, you will first need to know where the files are located on your local system.
+      </p>
+      <p>
+        To do so, navigate to the location of the files on your computer. This can be accomplished on a Mac by using the Finder application or on Windows with File Explorer application. Common locations for user data at the user's home directory, the Desktop and My Documents.
+      </p>
+      <p>
+        Once you have identified the location of the files, you can right-click on them and select either Get Info (on Mac) or Properties (on Windows) to view the path location on your local system.
+      </p>
+      <figure id="figure1">
+        <img src="https://a2cps-staging.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/6c/ea/6ceacba9-7824-4925-b924-dd317040b629/picture1.png__1170x0_q85_subsampling-2_upscale.png" />
+        <figcaption>
+          Figure 1. Use Get Info to determine “Where” the path of your data file(s) is
+        </figcaption>
+      </figure>
+
+      <p>
+        For example, a file located in a folder named <kbd>portal-data</kbd> under <code>Documents</code> would have the following path:
+      </p>
+      <dl>
+        <dt>On Mac</dt>
+        <dd><code>/Users/<kbd>username</kbd>/Documents/<kbd>portal-data</kbd>/my_file.txt</code></dd>
+        <dt>On Windows</dt>
+        <dd><code>\Users\<kbd>username</kbd>\My Documents\<kbd>portal-data</kbd>\my_file.txt</code></dd>
+      </dl>
+      <h4 id="storage_data_path">
+        Determining the Path to Your Transfer Directory
+      </h4>
+
+      <p>
+        A transfer directory on the remote storage server associated with the portal you are accessing it through will be established when your account is given access to the portal and completes the on-boarding procedure. The transfer directory path will be unique for every institution and project.
+      </p>
+      <details class="small"><summary>Examples:</summary>
+        <!-- TODO: Confirm example accuracy -->
+        {# SEE: https://github.com/TACC/Core-CMS/pull/152/files/89cd2a51d3b0904421df9438be2385f46c0f7ae6?file-filters%5B%5D=.html#r598831265 #}
+        <pre><code>/corral-repl/utexas/<kbd>project_name</kbd>/submissions/</code></pre>,
+        <pre><code>/corral-repl/uta/<kbd>project_name</kbd>/submissions/</code></pre>,
+        <pre><code>/corral-repl/utep/<kbd>project_name</kbd>/submissions/</code></pre>,
+        <pre><code>/corral-repl/uth/<kbd>project_name</kbd>/submissions/</code></pre>,
+        <pre><code>/corral-secure/projects/A2CPS/submissions/utaustin/</code></pre>
+        <blockquote>
+          <p>
+            Note: The transfer directory name will always be one word and in lowercase.
+          </p>
+        </blockquote>
+      </details>
```

Comment:
__Background__

1. Original Comment: https://github.com/TACC/Core-CMS/pull/152#discussion_r597287313
2. File was renamed.
2. Status Before File Rename: https://github.com/TACC/Core-CMS/pull/152#discussion_r598922463

__Questions__

1. Is this "portal agnostic enough"?
2. What should the other examples be? Or should there be none?

@rstijerina We look to you because you know the new paths better.

Reply:
My suggestion would be to simply remove all of these examples:

<!-- TODO: Confirm example accuracy -->
        {# SEE: https://github.com/TACC/Core-CMS/pull/152/files/89cd2a51d3b0904421df9438be2385f46c0f7ae6?file-filters%5B%5D=.html#r598831265 #}
        <pre><code>/corral-repl/utexas/<kbd>project_name</kbd>/submissions/</code></pre>,
        <pre><code>/corral-repl/uta/<kbd>project_name</kbd>/submissions/</code></pre>,
        <pre><code>/corral-repl/utep/<kbd>project_name</kbd>/submissions/</code></pre>,
        <pre><code>/corral-repl/uth/<kbd>project_name</kbd>/submissions/</code></pre>,
        <pre><code>/corral-secure/projects/A2CPS/submissions/utaustin/</code></pre>


Trying to anticipate "all the potential valid examples" seems to be a bridge too far.
The instruction tell them how to identify their target folder, so this is not really necessary.

Reply:
I am unaware of any of [these directory paths](https://github.com/TACC/Core-CMS/pull/152/files#diff-47c9d1edf459fa77665b59f2b6da00208d49d252138822304821e1880defa939R236-R240), besides `corral-secure` which only applies to A2CPS at the moment.

> Note: The transfer directory name will always be one word and in lowercase.

^ I'm also not sure what this is about.

Tracy should have better requirements on what this is supposed to achieve and who it's for.

Is the purpose for this guide to show people how to find the path to their Portal "Shared Workspace" directories (formerly "Projects" in CEPv1)? If so, it will depend on the portal like John mentioned, but most follow the scheme he supplied: `/corral-repl/tacc/aci/A2CPS/projects/A2CPS-##`

However, in CEPv2, we have the "View Path" button which tells the user the path - should we default to pointing the user there first?
<img width="1565" alt="Screen Shot 2021-03-22 at 1 17 06 PM" src="https://user-images.githubusercontent.com/20326896/112038514-f15be080-8b10-11eb-8942-822b052229d8.png">

Reply:
I have removed _both_ the dubious examples _and_ the potentially inaccurate statement about transfer directory name casing. _The statement "If you are unsure of your transfer directory path, please consult your project PI directly." solves the purpose of both._

Regarding "View Path", I am not keen on referencing Portal UI (which this document has not done) until we have a Portal User Guide, like FP-752 would provide (this ticket will be re-opened, for Frontera CMS). _But, I have added a succinct comment (hidden from users) to suggest this idea._

Reply:
Works for me 👍

Reply:
Ditto!